rotkehlchen/tests/unit/test_evm_misc.py

```diff
@@ -55,19 +54,3 @@ def test_weth_is_supported():
         set(CHAIN_ID_TO_WETH_MAPPING.keys()) ==
         set(typing.get_args(SUPPORTED_CHAIN_IDS)) - CHAINS_WITHOUT_NATIVE_ETH
     )
-
-
-def test_evminquirer_has_cachable_mixin(blockchain):
```

Comment:
Why is this removed? Has the problem been solved? You moved the logic to the manager and not the inquirer? So this is not needed?

Reply:
ah no, I thought it's testing something else

Reply:
so this was testing for `ensure_cache_data_is_updated`. But now thats the part of base EvmInquirer class. So, I'm changing it to check the `CurveManagerMixin` now